### ISSUE TITLE
Config files not read in gatling-maven-plugin  2.1.5

### ISSUE THREAD

**Issue by dibyom** [gatling/gatling-maven#4]
I upgraded to the plugin version 2.1.5. I am using a modified gatling.conf in the resources folder with the console data writer turned off. However, I still see console output. When I change the plugin version to 2.1.2, it seems to be working normally and I do not see any periodic console output.

**Comment by slandelle** [comment]
Do you run on Linux or Windows?
Which version of maven do you use?

**Comment by slandelle** [comment]
What's your set up exactly? Do you override configFolder? Do you have multiple gatling.conf (like one in default src/test/resources and one in custom configFolder)?

**Comment by dibyom** [comment]
I tried this on two environments: AWS Linux with maven 3.3.1 and on Mac OSX with maven 3.2.3 with identical results. 
I put my modified gatling.conf in the src/test/resources folder and and using the default configFolder path (src/test/resources).

**Comment by slandelle** [comment]
Then we can't reproduce. Could you please fork the [demo repo](https://github.com/gatling/gatling-maven-plugin-demo) and provide a reproducer, please?

**Comment by dibyom** [comment]
Here is my fork: https://github.com/dibyom/gatling-maven-plugin-demo
I modified the gatling.conf to disable console writer. I have two branches:
master has the latest version where we see console output even though I disabled it in the file.
working-version has an older version 2.0.3 (though 2.1.2 seems to work on my mac too).

Tried it on mac and AWS linux.

**Comment by slandelle** [comment]
I just ran your fork's master branch (which uses 2.2.0-M2) with `mvn clean verify`:
```
Simulation computerdatabase.BasicSimulation started...
Simulation finished
Parsing log file(s)...
Parsing log file(s) done
Generating reports...
================================================================================
---- Global Information --------------------------------------------------------
> request count                                         13 (OK=13     KO=0     )
> min response time                                    150 (OK=150    KO=-     )
> max response time                                    662 (OK=662    KO=-     )
> mean response time                                   214 (OK=214    KO=-     )
> std deviation                                        131 (OK=131    KO=-     )
> response time 50th percentile                        173 (OK=173    KO=-     )
> response time 75th percentile                        201 (OK=201    KO=-     )
> mean requests/sec                                   0.52 (OK=0.52   KO=-     )
---- Response Time Distribution ------------------------------------------------
> t < 800 ms                                            13 (100%)
> 800 ms < t < 1200 ms                                   0 (  0%)
> t > 1200 ms                                            0 (  0%)
> failed                                                 0 (  0%)
================================================================================
```

As you can see, console is disabled (the 5 sec periodic log, not the final statement).

Closing for now as I really can't reproduce.

Either we're not talking about the same thing, or there's something off with your environment.

**Comment by dibyom** [comment]
Could you tell me what environment you were running this on (OS and maven version) so I can try it there?

**Comment by slandelle** [comment]
OSX, JDK7, maven 3.3.1.

**Comment by slandelle** [comment]
Are you sure you don't have some MAVEN_OPTS or system properties that override the values defined in gatling.conf?

**Comment by dibyom** [comment]
Interesting. I ran it with mvn clean verify and it works. However, mvn gatling:execute does not work

**Comment by slandelle** [comment]
I think I get it.

With `gatling:execute`, you only run the task, so phases are bypassed, like the `test` one that would copy your "src/test/resources" content into "test-classes" so they get into the classpath.

And we fixed a bug in 2.1.5 where "src/test/resource" was added to the classpath, causing classpath duplicates (such as logback complaining about `logback.xml` duplicates).

The only solution I see is to stop launching the task directly, but launch the maven phase (`test` or `integration-test`).
